msg/RadarReturn.msg

```diff
@@ -0,0 +1,10 @@
+# All variables below are relative to the radar's frame of reference.
+# This message is not meant to be used alone but as part of a stamped or array message.
+
+float32 range                            # Distance (m) from the sensor to the detected point.
```

Comment:
Might need to change all uses of "point" to "return" now that the message name has changed.

Reply:
It's likely `return` is a reserved word.

Reply:
I'm only referring to the comments.

Reply:
done